src/components/calcite-input/calcite-input.tsx

```diff
@@ -70,6 +72,9 @@ export class CalciteInput {
     this.setDisabledAction();
   }
 
+  /** for number values, displays the locale's group separator */
+  @Prop() groupSeparator = false;
```

Comment:
nitpick: This could be an optional prop like: `@Prop() groupSeparator?: boolean;`